app/controllers/documents_controller.rb

```diff
@@ -67,9 +114,9 @@ def index
   # GET /documents/1
   # GET /documents/1.json
   def show
-    if request.path != document_path(@document)
-      redirect_to @document, status: :moved_permanently
-    end
+    # if request.path != document_path(@document)
```

Comment:
In general, unless there's a strong reason not to, let's remove commented out code. Not a blocker.